esm_runscripts/config_initialization.py

```diff
@@ -14,7 +14,7 @@ def init_first_user_config(command_line_config, user_config):
     # maybe switch to another runscript, if iterative coupling
     if user_config["general"].get("iterative_coupling", False):
         user_config = chunky_parts.setup_correct_chunk_config(user_config)
-        next_model = user_config["general"]["original_config"]["general"]["model_queue"][1]
+        next_model = user_config["general"]["original_config"]["general"]["model_queue"][-1]
```

Comment:
don't really understand this diff... shouldn't you just be rotating to the "next" model, instead of taking the last one?

Reply:
Yes, I did this when I experimented with two PISM instances, i.e. in total three models (awiesm, pism1, pism2) and it worked out better. But I guess it's better to leave it at the old value at the moment.

Reply:
What does `model queue` look like? is that a list? Because at some point (when we have vilma, pism, and climate all at once) we will have three models. So here we need a way to rotate around each model, something like:
```python
# Assume you have
model_queue = ["awicm", "pism", "vilma"]
current_model = user_config['general']['original_config']['general']['model_queue'].pop()
# Now model queue is something like ["pism", "vilma"] and current model is "awicm"
user_config['general']['original_config']['general']['model_queue'].append(current_model)
# Now model queue is:
model_queue = ["pism", "vilma", "awicm"]
```

in all likelihood there is a python build-in function to do this, but that is how I would solve it. I need more context though to know that is what you actually want to do here...